README.md

```diff
@@ -71,15 +71,15 @@ client.fetch_marc_hash(instance_hrid: "a7927874")
   [{"003"=>"FOLIO"}....]
   }
 
-# Import a MARC record into FOLIO
-data_importer = client.data_import(marc: my_marc, job_profile_id: '4ba4f4ab', job_profile_name: 'ETDs')
+# Import a MARC file into FOLIO
+data_importer = client.data_import(file_path:: "marc_file.mrc", job_profile_id: '4ba4f4ab', job_profile_name: 'ETDs')
```

Comment:
As part of the work to update how to determine when a file's done processing, align the Ruby client approach with our Python client approach: take a file (with one or more records) instead of an in-memory MARC record.

Reply:
Why? For the uses of this library, there might not be a file on disk.

Reply:
@justinlittman To take an even further step back: what are our known use cases for data import in rubyland?